HW2/P5/driver.py

```diff
@@ -80,3 +103,4 @@ def randcolor():
             # SUBPROBLEM 3: sort objects by location.  Be sure to update the
             # grid if objects' indices change!  Also be sure to sort the
             # velocities with their object positions!
+            # at first, we create a
```

Comment:
OMG, I must have been too excited when I made the pull request... and I accidentally deleted my sorting part..Jesus..I pasted my code here, so I hope our grader could go to my P5.driver commit and looked for the second last commit, you will find this code.. I am sorry for my stupidity 
# SUBPROBLEM 3: sort objects by location.  Be sure to update the
             # grid if objects' indices change!  Also be sure to sort the
             # velocities with their object positions!
             # at first, we create a grid loc
             grid_loc = (positions/grid_spacing).astype(int)
             morton = map(lambda a,b: cmp_zorder(a,b), grid_loc, np.array(list(grid_loc[1:])+list(grid_loc[:1])))
             grid[:] = -1
             grid_index = np.argsort(morton)
             velocities, positions= velocities[grid_index], positions[grid_index]
 
             #let's update our grid values
             for i in range(num_balls):
                 if (positions[i,0] >= 0) and (positions[i,1] >= 0) and (positions[i,0] <= grid_size) and (positions[i,1] <= grid_size):
                     grid[(positions[i,0] / grid_spacing).astype(int),
-                         (positions[i,1] / grid_spacing).astype(int)] = i
-
-
-
+                         (positions[i,1] / grid_spacing).astype(int)] = i